multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -39,6 +39,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("Aqua Manga", "https://aquamanga.com", "en", overrideVersionCode = 7),
         SingleLang("AQUA Scans", "https://aquascans.com", "en", className = "ManhwaWorld", overrideVersionCode = 1),
         SingleLang("ArazNovel", "https://www.araznovel.com", "tr", overrideVersionCode = 3),
+        SingleLang("Arcanescans", "https://arcanescans.com/", "en"),
```

Comment:
Don't include trailing slashes.

```suggestion
        SingleLang("Arcanescans", "https://arcanescans.com", "en"),
```